tests/unit/utils/test_utils.py

```diff
@@ -0,0 +1,74 @@
+from csodiaq.utils import (
+    create_outfile_header,
+)
+import pandas as pd
+import pytest
+
+
+@pytest.fixture
+def outputDirectory():
+    return "test/output/dir"
```

Comment:
For temporary file outputs, try to use the `tempfile` module to avoid cluttering the filesystem:  
```Python
import tempfile
t_dir = tempfile.TemporaryDirectory(prefix="csodiaq")
t_file = tempfile.NamedTemporaryFile(prefix="csodiaq")
```  
  
Note that `TemporaryDirectory` and `NamedTemporaryFile` clean up after themselves; I'm not sure whether they'll play nice with `@pytest.fixture`. If those don't work, you can use `mkstemp` instead, but that requires that you delete it afterwards.

Reply:
No actual files are created using this fixture - it's more for creating an output file name when an output directory path is provided. If I wind up needing to create a file, I will turn to using tempfile here.